docker/Makefile

```diff
@@ -76,6 +87,21 @@ vscode-image:  qemu-image | check-docker
 		vscode
 
 ### docker run targets ###
+.PHONY: format
+TARGET += format
+HELP_idf = runs formatting container
+format: | format-image check-docker
+	docker run \
+		--rm \
+		-ti \
+		$(DOCKER_DEV) \
+		-v format-home:/home/developer:rw \
```

Comment:
is a persistent home needed?

Reply:
As we want to change our src files, i think so

Reply:
I don't see anything that is created in the home directory,
therefore I don't the necessity

Reply:
the c and header files are adapted to the style, nothing is created

Reply:
the files this container modifies are mounted and have nothing to do with the container home dir. the other docker containers feature a persistent home for custom configs and caches of the toolchain (i.e. ccache, .gdbinit, ...). this is not needed in this case (why customize the format style?), hence my initial question.